array/array.go

```diff
@@ -5,11 +5,7 @@ func countMap(arrs ...[]string) (result map[string]int) {
 	for _, arr := range arrs {
 		distinctArr := DistinctString(arr)
 		for _, value := range distinctArr {
-			if _, ok := result[value]; ok {
-				result[value]++
-			} else {
-				result[value] = 1
-			}
+			result[value]++
```

Comment:
Didn't know you could do this, nice!

Reply:
The linter told me ^^

Reply:
default value when reading from a map is always the zero value, which in the case of an integer is zero